src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,133 @@
-package core.basesyntax;
-
-public class MyHashMap<K, V> implements MyMap<K, V> {
-
-    @Override
-    public void put(K key, V value) {
-
-    }
-
-    @Override
-    public V getValue(K key) {
-        return null;
-    }
-
-    @Override
-    public int getSize() {
-        return 0;
-    }
-}
+package core.basesyntax;
+
+import java.util.Objects;
+
+public class MyHashMap<K, V> implements MyMap<K, V> {
+
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private int capacity;

```

Comment:
Please use length of array instead of capacity. More variables used - more room for mistake
```suggestion
```